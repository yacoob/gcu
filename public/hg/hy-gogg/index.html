<html lang='en'>
<head>
  <meta charset='utf-8'>
  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
  <meta name='viewport' content='width=device-width, initial-scale=1'>
  <!--[if lt IE 9]>
  <script src='https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js'></script>
  <script src='https://oss.maxcdn.com/respond/1.4.2/respond.min.js'></script>
  <![endif]-->
  <link href='/css/bootstrap.min.css' rel='stylesheet'>
  <link href='/css/gcu.css' rel='stylesheet'>
  <link href='/i/favicon.ico' rel='icon' type='image/x-icon'/>
  <link href='//syn.tactical-grace.net/f/full/gcu-00622.jpg' rel='image_src'/>
  <meta property='og:image' content='https://syn.tactical-grace.net/f/full/gcu-00622.jpg'/>
  <meta name='description' content='tactical grace presents: Hy-Gogg'/>
  <link href='http://gcu.tactical-grace.net/hg/hy-gogg/' rel='canonical'/>
  <link rel='alternate' type='application/rss+xml' title='tactical grace feed' href='/index.xml'>
  <title>HG: Hy-Gogg | tactical grace</title>
  <meta content='HG: Hy-Gogg | tactical grace' itemprop='name'/>
</head>
<body>
<div class='container'>
<div class='row'><!-- title and mininav -->
  <div class='col-sm-12 col-md-8 text-right'>
  <h1 class='title'><a href='/hg/hy-gogg/'>HG: Hy-Gogg</a></h1>
</div> <!-- column -->
  <div class='col-sm-12 col-md-4 text-right voffset-top'>
    <a href='/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-home'></span></button></a>
    <a href='/hg/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-th'></span></button></a>
    <a href='/hg/guntank/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-chevron-left'></span></button></a>
    <a href='/hg/juaggu/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-chevron-right'></span></button></a>
  </div>
</div> <!-- row -->
  <div class='row voffset-top'>
    <h2 id='2015-01-25'>2015-01-25</h2>
  </div>
<div class='row voffset-small'>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00613.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00613.jpg' title='Three and a half runners.' alt='Three and a half runners.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00614.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00614.jpg' title='Arm parts.' alt='Arm parts.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00615.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00615.jpg' title='Arm assembled.' alt='Arm assembled.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00616.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00616.jpg' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00617.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00617.jpg' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00618.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00618.jpg' title='Big Hy-Gogg, Crabtriya&#39;s cousin.' alt='Big Hy-Gogg, Crabtriya&#39;s cousin.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00619.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00619.jpg' title='Flappy feet.' alt='Flappy feet.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00620.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00620.jpg' title='2 ∞ &amp; → !' alt='2 ∞ &amp; → !' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00621.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00621.jpg' title='&#34;BUT I&#39;M HELPING!&#34;' alt='&#34;BUT I&#39;M HELPING!&#34;' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00622.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00622.jpg' title='Chubby Superman wannabe.' alt='Chubby Superman wannabe.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00623.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00623.jpg' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
</div>
<div class='row'>
  <div class='col-sm-12 col-md-7 col-md-offset-1 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-default' href='/hg/' role='button'>
        <span class='glyphicon glyphicon-pawn hidden-xs'></span>
        HG</a>
      <a class='btn btn-default' href='/rg/' role='button'>
        <span class='glyphicon glyphicon-knight hidden-xs'></span>
        RG</a>
      <a class='btn btn-default' href='/mg/' role='button'>
        <span class='glyphicon glyphicon-bishop hidden-xs'></span>
        MG</a>
      <a class='btn btn-default' href='/pg/' role='button'>
        <span class='glyphicon glyphicon-king hidden-xs'></span>
        PG</a>
      <a class='btn btn-default' href='/varia/' role='button'>
        <span class='glyphicon glyphicon-tower hidden-xs'></span>
        Varia</a>
    </div> <!-- btn-group -->
  </div> <!-- column -->
  <div class='col-sm-12 col-md-4 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-danger btn-lg' href='/everything/' role='button'>
        <span class='glyphicon glyphicon-fire'></span>
        Show me everything!</a>
    </div>
  </div> <!-- column -->
</div> <!-- row -->
</div> <!-- container -->
<link href='http://fonts.googleapis.com/css?family=Exo' rel='stylesheet' type='text/css'>
<script> <!-- analytics -->
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-35396418-1', 'auto');
  ga('send', 'pageview');
</script>
<link type='text/css' rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/lightgallery/1.3.6/css/lightgallery.min.css' />
<!-- include lightbox js/css here -->
<script src='https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.0/jquery.min.js'></script>
<script src='https://cdnjs.cloudflare.com/ajax/libs/lightgallery/1.3.6/js/lightgallery.min.js'></script>
<script src='/js/lg-fullscreen.min.js'></script>
<script src='/js/lg-thumbnail.min.js'></script>
<script src='/js/lg-autoplay.min.js'></script>
<script src='/js/lg-zoom.min.js'></script>
<script src='/js/lg-video.min.js'></script>
<script src='/js/gcu.js'></script>
<script type='text/javascript'>
$(document).ready(gcu.postPageHandler);
</script>
</body></html>