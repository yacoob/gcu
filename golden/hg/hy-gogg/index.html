<html lang='en'>
<head>
  <meta charset='utf-8'>
  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
  <meta name='viewport' content='width=device-width, initial-scale=1'>
  <link href='https://cdn.jsdelivr.net/npm/bootstrap3@3/dist/css/bootstrap.min.css' rel='stylesheet'>
  <link href='/css/gcu.css' rel='stylesheet'>
  <link href='/i/favicon.ico' rel='icon' type='image/x-icon'/>
  <link href='/photos/full/gcu-00622.jpg' rel='image_src'/>
  <meta property='og:image' content='https://gcu.tactical-grace.net/photos/full/gcu-00622.jpg'/>
  <meta name='description' content='tactical grace presents: Hy-Gogg'/>
  <link href='https://gcu.tactical-grace.net/hg/hy-gogg/' rel='canonical'/>
  <link rel='alternate' type='application/rss+xml' title='tactical grace feed' href='/index.xml'>
  <title>HG: Hy-Gogg | tactical grace</title>
  <meta content='HG: Hy-Gogg | tactical grace' itemprop='name'/>
</head>
<body>
<div class='container'>
<div class='row'><!-- title and mininav -->
  <div class='col-sm-12 col-md-8 text-right'>
  <h1 class='title'><a href='/hg/hy-gogg/'>HG: Hy-Gogg</a></h1>
</div> <!-- column -->
  <div class='col-sm-12 col-md-4 text-right voffset-top'>
    <a data-toggle='tooltip' title='main page' href='/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-home'></span></button></a>
    <a data-toggle='tooltip' title='contact the author' href='https://goo.gl/forms/iJEsfLjBy1FE86aQ2'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-envelope'></span></button></a>
    <a data-toggle='tooltip' title='grade page' href='/hg/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-th'></span></button></a>
    <a data-toggle='tooltip' title='previous kit' href='/hg/guntank/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-chevron-left'></span></button></a>
    <a data-toggle='tooltip' title='next kit' href='/hg/juaggu/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-chevron-right'></span></button></a>
  </div>
</div> <!-- row -->
  <div class='row voffset-top'>
    <h2 id='2015-01-25'>2015-01-25</h2>
  </div>
<div class='row voffset-small'>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00613.jpg' data-sub-html='Three and a half runners.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00613.jpg' alt='Three and a half runners.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00614.jpg' data-sub-html='Arm parts.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00614.jpg' alt='Arm parts.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00615.jpg' data-sub-html='Arm assembled.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00615.jpg' alt='Arm assembled.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00616.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00616.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00617.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00617.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00618.jpg' data-sub-html='Big Hy-Gogg, Crabtriya&#39;s cousin.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00618.jpg' alt='Big Hy-Gogg, Crabtriya&#39;s cousin.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00619.jpg' data-sub-html='Flappy feet.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00619.jpg' alt='Flappy feet.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00620.jpg' data-sub-html='2 ∞ &amp; → !'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00620.jpg' alt='2 ∞ &amp; → !' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00621.jpg' data-sub-html='&#34;BUT I&#39;M HELPING!&#34;'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00621.jpg' alt='&#34;BUT I&#39;M HELPING!&#34;' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00622.jpg' data-sub-html='Chubby Superman wannabe.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00622.jpg' alt='Chubby Superman wannabe.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00623.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00623.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
</div>
<div class='row'>
  <div class='col-sm-12 col-md-7 col-md-offset-1 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-default' href='/hg/' role='button'>
        <span class='glyphicon glyphicon-pawn hidden-xs'></span>
        HG</a>
      <a class='btn btn-default' href='/rg/' role='button'>
        <span class='glyphicon glyphicon-knight hidden-xs'></span>
        RG</a>
      <a class='btn btn-default' href='/mg/' role='button'>
        <span class='glyphicon glyphicon-bishop hidden-xs'></span>
        MG</a>
      <a class='btn btn-default' href='/pg/' role='button'>
        <span class='glyphicon glyphicon-king hidden-xs'></span>
        PG</a>
      <a class='btn btn-default' href='/varia/' role='button'>
        <span class='glyphicon glyphicon-tower hidden-xs'></span>
        Varia</a>
    </div> <!-- btn-group -->
  </div> <!-- column -->
  <div class='col-sm-12 col-md-4 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-danger btn-lg' href='/everything/' role='button'>
        <span class='glyphicon glyphicon-fire'></span>
        Show me everything!</a>
    </div>
  </div> <!-- column -->
</div> <!-- row -->
</div> <!-- container -->
<link href='https://fonts.googleapis.com/css?family=Exo' rel='stylesheet' type='text/css'>
<script> <!-- analytics -->
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-35396418-1', 'auto');
  ga('send', 'pageview');
</script>
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/lightgallery.js@1/dist/css/lightgallery.min.css" />
<script src="https://cdn.jsdelivr.net/npm/lightgallery.js@1/dist/js/lightgallery.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-thumbnail.js@1/dist/lg-thumbnail.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-zoom.js@1/dist/lg-zoom.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-video.js@1/dist/lg-video.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-fullscreen.js@1/dist/lg-fullscreen.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-autoplay.js@1/dist/lg-autoplay.min.js"></script>
<script src='/js/gcu.js'></script>
<script type='text/javascript'>
document.addEventListener('DOMContentLoaded', gcu.postPageHandler);
</script>
</body></html>